packages/jest-environment-jsdom/src/index.ts

```diff
@@ -119,6 +119,7 @@ class JSDOMEnvironment implements JestEnvironment {
       }
     }
     this.errorEventListener = null;
+    // @ts-ignore
     this.global = null;
```

Comment:
It should be nullable

Reply:
I think we don't need set global to null because of we just teardown for Window APIs from JSDOM! I will remove this line.